packages/datadog-plugin-cypress/src/plugin.js

```diff
@@ -69,12 +70,15 @@ module.exports = (on, config) => {
       return null
     },
     'dd:afterEach': (test) => {
-      const { state, error } = test
+      const { state, error, ...additionalSpans } = test
```

Comment:
is `...aditionalSpans` ever not empty? There's nothing being passed in the `cy.task` call

Reply:
In most cases it would be empty.
However utilising these allows me to add my own `dd:afterEach` or `dd:beforeEach`.
Instead of:

```
require('dd-trace/cypress/support')
```

I can write something like this

```
beforeEach(() => {
  const { team, device } = getTestDetails(Cypress.mocha.getRunner().suite.ctx.currentTest.id);

  cy.task('dd:beforeEach', {
    testName: Cypress.mocha.getRunner().suite.ctx.currentTest.fullTitle(),
    testSuite: Cypress.mocha.getRootSuite().file,
    'test.team': team,
    'test.device': device,
  });
});
```

Reply:
that makes sense. At that point, though, I think it'd be better to name the task `dd:setTags`